generators/cleanup.js

```diff
@@ -75,6 +77,29 @@ function cleanupOldFiles(generator) {
         generator.removeFile(`${ANGULAR_DIR}admin/metrics/metrics-modal.component.ts`);
         generator.removeFile(`${CLIENT_TEST_SRC_DIR}spec/app/admin/metrics/metrics-modal.component.spec.ts`);
     }
+    if (generator.isJhipsterVersionLessThan('6.3.0')) {
+        generator.removeFile(`${ANGULAR_DIR}account/index.ts`);
+        generator.removeFile(`${ANGULAR_DIR}admin/index.ts`);
+        generator.removeFile(`${ANGULAR_DIR}core/index.ts`);
+        generator.removeFile(`${ANGULAR_DIR}home/index.ts`);
+        generator.removeFile(`${ANGULAR_DIR}layouts/index.ts`);
+        generator.removeFile(`${ANGULAR_DIR}shared/index.ts`);
+        if (generator.withEntities) {
```

Comment:
I think entity related cleanup should be handled in `entity` sub-generator. Please check `entity/index.js`

Reply:
Yes you're right! When doing PR I couldn't find suitable place in `entity` sub-generator, but now I found `cleanup()` method in `entity` sub-generator and will move entity cleaning into this function.

Reply:
Change done.